.semaphore/semaphore.yml

```diff
@@ -0,0 +1,18 @@
+version: v1.0
+name: Test
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: Test
+    task:
+      jobs:
+        - name: Test
+          commands:
+            - checkout
+            - sem-version node 12
```

Comment:
Minor thing, but we should probably run this on node 14. 12 has moved to maintenance and 14 is now the current LTS release.

Reply:
Good point! Thanks